CHANGELOG.md

```diff
@@ -6,14 +6,41 @@
 
-    Thank you to @Arkham for bringing this to our attention!
+    Thank you to [@Arkham](https://github.com/Arkham) for bringing this to our attention!
 
 
-  * **Bugfix: Fix memory leak in the Curb instrumentation
+  * **Bugfix: Fix memory leak in the Curb instrumentation**
 
-    With [PR #1518](https://github.com/newrelic/newrelic-ruby-agent/pull/1518) community member [@charkost](https://github.com/charkost) was able to rework the `on_failure` callback logic prepped via the agent's Curb instrumentation in order to avoid some nesting that was causing memory leaks.
+    Community member [@charkost](https://github.com/charkost) was able to rework the `on_failure` callback logic prepped via the agent's Curb instrumentation in order to avoid some nesting that was causing memory leaks. [PR#1518](https://github.com/newrelic/newrelic-ruby-agent/pull/1518)
+
+    Many thanks for both the heads up on the issue and the fix, [@charkost](https://github.com/charkost)!
+
+
+  * **Cleanup: Performance tests, constants, rubocop-minitest assertions and refutations**
+
+    Community member [@esquith](https://github.com/esquith) contributed a whole slew of cleanup successes for our performance test configuration, orphaned constants in our code base, and RuboCop related improvements. [PR#1406](https://github.com/newrelic/newrelic-ruby-agent/pull/1406) [PR#1408](https://github.com/newrelic/newrelic-ruby-agent/pull/1408) [PR#1409](https://github.com/newrelic/newrelic-ruby-agent/pull/1409) [PR#1411](https://github.com/newrelic/newrelic-ruby-agent/pull/1411)
+
+    Thank you [@esquith](https://github.com/esquith) for these great contributions!
+
+
+  * **CI: nofify on a change from failure to success**
+
+    A super handy, much beloved feature of certain CI and build systems is to not only notify when builds start to fail, but also to notify again when the builds once again start to go green. Community member [@luigieai](https://github.com/luigieai) was able to figure out how configure our existing complex, multiple-3rd-party-action based GitHub Actions pipeline to notify on a switch back to success from failure. [PR#1519](https://github.com/newrelic/newrelic-ruby-agent/pull/1519)
+
+    This is much appreciated! Thank you, [@luigieai](https://github.com/luigieai).
+
+
+  * **Spelling corrections**
+
+    Community member [@jsoref](https://github.com/jsoref), author of the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action, contributed a significant number of spelling corrections throughout the code base. The intelligent issues that were flagged made for a more comprehensive once over than a simple dictionary based check would have been able to provide, and the changs are much appreciated. [PR#1508](https://github.com/newrelic/newrelic-ruby-agent/pull/1508)
```

Comment:
.... and the `changes` are much appreciated

Reply:
addressed with [24855d4](https://github.com/newrelic/newrelic-ruby-agent/pull/1530/commits/24855d46ce9a7901a55021ea6830b328a476fec6)